schema/message_fmt2/testgen_schema.json

```diff
@@ -67,30 +66,24 @@
             }
           }
         },
-        {
-          "properties": {
-            "defaultTestProperties": {
-              "required": [
-                "srcs"
-              ]
-            }
-          }
-        },
         {
           "properties": {
              "tests": {
-               "type": "array",
-               "items": {
-                "required": [
-                  "src"
-                ]
+               "type": "string",
```

Comment:
This "type" should always be "array" because it is describing the "tests" property.
(Since "items" is already a child, I suppose "type" is technically not required and can be removed if it's misleading). 

I think we only need this:
```jsonc
// ...
    {
      "anyOf": [
        {
          "properties": {
            "defaultTestProperties": {
              "required": [
                "src"
              ]
            }
          }
        },
        {
          "properties": {
            "tests": {
              "type": "array",  // For clarity only
              "items": {
                "required": [
                  "src"
                ]
              }
            }
          }
        }
      ]
    },
// ...
```

and the "src" string/array definition is all handled under "$defs".

Reply:
Fixed -- I also noticed that the second "properties" clause in the "anyOf" was duplicated for some reason (probably my mistake), so I removed that.